### PR TITLE
feat: jbang init --edit to have editor open directly

### PR DESCRIPTION
**Pull request by maxandersen**

### CODE REVIEW

**Comment by maxandersen** [comment]
quick'n'dirty...

File: src/main/java/dev/jbang/cli/Init.java [hunk 1]
```diff
@@ -116,13 +119,22 @@ public Integer doCall() throws IOException {
...
-				+ "' or edit it using 'jbang edit --open=[editor] "
-				+ renderedScriptOrFile + "' where [editor] is your editor or IDE, e.g. '"
-				+ Edit.knownEditors[new Random().nextInt(Edit.knownEditors.length)]
-				+ "'. If your IDE supports JBang, you can edit the directory instead: 'jbang edit . '"
-				+ renderedScriptOrFile + ". See https://jbang.dev/ide");
-
+		if (edit) {
+			info("File initialized. Opening editor for you. You can also now run it with 'jbang "
+					+ renderedScriptOrFile);
+			// TODO: quick hack that gets the job of opening editor done; but really should
```
Comment (quintesse) [inline src/main/java/dev/jbang/cli/Init.java:125]: To me this actually isn't a hack but a completely valid way of doing things. An API _might_ be nice but could very well be overkill for something that (so far at least) isn't meant to be used as a library.

**Review by quintesse — APPROVED** [review]
Love it :-)

File: src/main/java/dev/jbang/cli/Init.java (hunk 1 above)
Comment (maxandersen) [reply src/main/java/dev/jbang/cli/Init.java:125]: its just thtat it prints out you can run with -b which is not the case for this ;)

**Pull request merged**